.travis.yml

```diff
@@ -18,19 +19,18 @@ script:
 
 
 before_deploy:
-  - mkdir ~/bin
+  - mkdir -p ~/.local/bin
```

Comment:
`~/.local/bin` is where `pip install --user` installs `awscli`. I figured it would be easier to install everything there rather than add two locations to the `PATH`.